app/templates/events/view/settings/options.hbs

```diff
@@ -0,0 +1,58 @@
+<div class="row">
```

Comment:
rename file name danger-zone.hbs

Reply:
I think not required as per issue's requirement